be/src/olap/tablet_schema.cpp

```diff
@@ -492,21 +491,14 @@ bool TabletColumn::is_row_store_column() const {
     return _col_name == BeConsts::ROW_STORE_COL;
 }
 
```

Comment:
why revert pr #19969？

Reply:
Thanks for the heads-up. That was my bad. This PR would have caused BE core, so I reverted it during testing. It's all fixed now.